packages/wallet/src/services/ProviderTracker/ProviderStatusTracker.ts

```diff
@@ -74,6 +76,7 @@ export interface ProviderStatusTrackerInternals {
 export const createProviderStatusTracker = (
   dependencies: ProviderStatusTrackerDependencies,
   { consideredOutOfSyncAfter }: ProviderStatusTrackerProps,
+  logger: Logger, // REVIEW: maybe we should transform this into object params instead of positional
```

Comment:
REVIEW comment here.
We should probably stop using positional params here. If the debugger logs solution is ok, I can update this code to use named params.

Reply:
Yes, it could be part of `ProviderStatusTrackerDependencies`

Reply:
Done in [3eeba2a](https://github.com/input-output-hk/cardano-js-sdk/pull/379/commits/3eeba2a461134843d202116d39c1edec2c118116)